src/components/Input/Input.vue

```diff
@@ -134,6 +137,18 @@ export default defineComponent({
     pattern: {
       type: String,
       required: false
+    },
+    autocomplete: {
+      type: Boolean,
```

Comment:
This should be `type: string`; https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete